mailbox/backup/pom.xml

```diff
@@ -69,13 +69,26 @@
             <scope>test</scope>
         </dependency>
         <dependency>
-            <groupId>com.google.guava</groupId>
```

Comment:
As we end up with no code in mailbox/backup, then there should be no change in dependencies, isn't it?

Reply:
I think it's the order of dependencies that changed. The package **google.guava** is now declared below all the artifacts from james group. It is our practice, no?

Reply:
How about the addition of james-server-data-api james-server-data-memory mockito-core (without test scope) ?